english/see-smore-options/see-smore-options.md

```diff
@@ -0,0 +1,6 @@
+Not sure of what to do, you decide to phone-a-friend for help.  You hit speed dial for your friend, Graham Hershey.  Graham picks up after three rings, but you sense this is not going to end well.  Graham sounds like he's already toasted, and the background noise suggests he's sitting in a bar.
+
+Do you:
+
+[Melt into tears and faint?](../pinch/help/faint/faint.md)
+[Sit back down, and go to sleep?](../sleep/more-sleep/more-sleep.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.